modules/showbook.js

```diff
@@ -0,0 +1,32 @@
+import Book from './book.js';
+
+export default class ShowBooks extends Book {
+  displayBook(book) {
+    const bookCard = document.createElement('div');
+    bookCard.classList.add('book-card');
+
+    const pTitle = document.createElement('p');
+    pTitle.textContent = `"${book.title}" by ${book.author}`;
+
+    const removeBtn = document.createElement('button');
+    removeBtn.textContent = 'Remove';
+    removeBtn.classList.add('btn', 'btn-outline-danger');
+
+    bookCard.append(pTitle, removeBtn);
+    this.booksContainer.appendChild(bookCard);
+
+    removeBtn.addEventListener('click', () => {
+      this.removeBook(book.title, book.author);
+      this.storeBook(this.books);
+      this.booksContainer.removeChild(bookCard);
+    });
+  }
+
+  iterateBooks() {
+    this.books = JSON.parse(localStorage.getItem('data-books'));
+    this.books.map((book) => this.displayBook(book));
+  }
+}
```

Comment:
- [ ] Dear @jugosack, Currently your functionality breaks when I try to add a new book (`the console mentions it fails to read a map method on a null object`),  Please address this for a fully functioning app. 


https://user-images.githubusercontent.com/69077061/212993960-29b33e21-6572-4c10-a500-6dfe672a55ef.mp4